### ISSUE TITLE
Out of date, broken on recent ubuntu releases

### ISSUE THREAD

**Issue by plang85** [MicrosoftDocs/azure-devops-docs#8949]
[Enter feedback here]
This workflow (`artifacts-keyring`) seems broken for users of recent Ubuntu releases - please could someone look into the github issues. Also `artifacts-keyring` has been in preview or at least a year without any updates.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c6cca7b9-7fe5-bc14-c7da-08983e920f81
* Version Independent ID: 7d8cf191-b229-0e2e-8cb3-01a80c4842fd
* Content: [Get started with Python packages - Azure Artifacts](https://docs.microsoft.com/en-us/azure/devops/artifacts/quickstarts/python-packages?view=azure-devops)
* Content Source: [docs/artifacts/quickstarts/python-packages.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/master/docs/artifacts/quickstarts/python-packages.md)
* Product: **devops**
* Technology: **devops-artifacts**
* GitHub Login: @ramiMSFT
* Microsoft Alias: **rabououn**

**Comment by ramiMSFT** [comment]
@herenhuang Can you take a look?

**Comment by herenhuang** [comment]
investigating!

**Comment by herenhuang** [comment]
Hey Phillip, we did take a look at the noted issues with the latest Ubuntu versions. Unfortunately we cannot commit to a fix date at the moment, though when we are able to we will make notice of it via the public ADO roadmap as well as on the Github issues themselves. 

Closing - though perhaps we can add a note in our docs on our knowledge of issues with artifacts-keyring on new Ubuntu versions, and to point folks to creating an item on the Github Issues board for them.
